app/src/main/java/com/recommend_tour/api/DataApiService.kt

```diff
@@ -0,0 +1,38 @@
+package com.recommend_tour.api
+
+import com.google.gson.GsonBuilder
+import com.recommend_tour.data.ApiResponse
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import okhttp3.logging.HttpLoggingInterceptor.Level
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+private const val MOBILE_APP = "recommend_tour"
+private const val SERVICE_KEY = "Lv3KHPJYHUvfCs0PRMXwrxDSUYHp3ePlZLPSbQ3JeGgai6UhMyWmBGFw79liTjnNd3pJaqRdcesC+1XWjnnTuA=="
```

Comment:
```suggestion
private const val SERVICE_KEY = "Lv3KHPJYHUvfCs0PRMXwrxDSUYHp3ePlZLPSbQ3JeGgai6UhMyWmBGFw79liTjnNd3pJaqRdcesC+1XWjnnTuA=="
```
API key값인듯한데 코드상에 이런식으로 노출하는것은 좋지않습니다.